samples/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Workaround for #261 which requires the bin and obj directory be in the project directory
+bin/
+obj/
```

Comment:
Can we just leave this alone in the .gitignore instead? I don't see how this helps.

Reply:
sure